addons/joda-time/src/test/scala/com/github/tminglei/slickpg/PgDateSupportJodaSuite.scala

```diff
@@ -145,6 +145,10 @@ class PgDateSupportJodaSuite extends AnyFunSuite with PostgresContainer {
           Datetimes.filter(_.id === 101L.bind).map(r => r.datetime.trunc("day")).result.head.map(
             r => assert(LocalDateTime.parse("2001-01-03T00:00:00") === r)
           ),
+          // dateBin
+         DBIO.seq(Option.when(pgVersion.take(2).toInt >= 14)(Datetimes.filter(_.id === 101L.bind).map(r => r.datetime.dateBin("1 hour", LocalDateTime.parse("2001-01-03T18:35:17"))).result.head.map(
```

Comment:
This could be refactored out for a more principled approach to supporting features that're only introduced in later versions of postgres, but I figured I'd test the waters first, and don't currently need any other feature missing from the existing set

Reply:
It doesn't matter, I think, since it's `postgresql` that really supports these functions, instead of `slick-pg`.
In fact, `slick-pg` is just a tool that translates from scala functions to pg functions.